src/hub.js

```diff
@@ -266,10 +266,10 @@ let routerBaseName = document.location.pathname
   .join("/");
 
 if (document.location.pathname.includes("hub.html")) {
-  routerBaseName = "";
+  routerBaseName = "/";
 }
 
-const history = createBrowserHistory({ basename: routerBaseName });
+const history = routerBaseName === "/" ? createMemoryHistory() : createBrowserHistory({ basename: routerBaseName });
```

Comment:
comment probably worthwhile here, this could be confusing for people who forget that we offer the default room now